scapy/contrib/gtp_v2.py

```diff
@@ -1291,6 +1303,19 @@ class IE_EPCO(gtp.IE_Base):
                                    length_from=lambda pkt: pkt.length - 1)]
 
 
+class IE_APCO(gtp.IE_Base):
+    name = "IE Additional Protocol Configuration Options"
+    fields_desc = [ByteEnumField("ietype", 163, IEType),
+                   ShortField("length", None),
+                   BitField("CR_flag", 0, 4),
+                   BitField("instance", 0, 4),
+                   BitField("Extension", 0, 1),
```

Comment:
Use lower case for field names wherever possible. Thanks.

Reply:
Done, changed Extension to lowercase. Did not change other fields to keep the same formatting between the IEs